test/dlc_tests/sanity/test_ecr_scan.py

```diff
@@ -28,6 +29,7 @@
     ECR_ENHANCED_REPO_REGION,
     is_generic_image,
     get_allowlist_path_for_enhanced_scan_from_env_variable,
+    get_ecr_scan_allowlist_path,
```

Comment:
nit: use a generic core function which gets the path prefix, then use the specific function to get the specific file path